analysis/tvla.py

```diff
@@ -234,41 +235,72 @@ def run_tvla(ctx: typer.Context):
     if cfg["test_type"] == "GENERAL_DATA":
         general_test_key = False
         general_test_data = True
-        specific_test = False
+        specific_test_byte = False
+        specific_test_bit = False
     elif cfg["test_type"] == "SPECIFIC_BYTE":
         general_test_key = False
         general_test_data = False
-        specific_test = True
+        specific_test_byte = True
+        specific_test_bit = False
+    elif cfg["test_type"] == "SPECIFIC_BIT":
+        general_test_key = False
+        general_test_data = False
+        specific_test_byte = False
+        specific_test_bit = True
     else:
         assert cfg["test_type"] == "GENERAL_KEY"
         general_test_key = True
         general_test_data = False
-        specific_test = False
+        specific_test_byte = False
+        specific_test_bit = False
 
     general_test = general_test_key or general_test_data
+    specific_test = specific_test_byte or specific_test_bit
 
     aes_num_rnds = 11
     aes_num_bytes = 16
+    aes_num_bits = 128
```

Comment:
These numbers are actually referring to the block size, rather than key size.
I've now renamed these variables to `aes_block_size_bytes` and `aes_block_size_bits` to remove confusion.